Tasks/AzureSpringCloudV0/task.json

```diff
@@ -18,8 +18,8 @@
   "preview": true,
   "version": {
     "Major": 0,
-    "Minor": 227,
-    "Patch": 1
+    "Minor": 230,
+    "Patch": 2
```

Comment:
Should this Patch version be 0? Because we advanced the Minor version and this is the first build of this minor version.